cli/blockchain_info/cli.py

```diff
@@ -0,0 +1,100 @@
+"""
+Provide implementation of the command line interface's batch commands.
+"""
+import sys
+import re
+
+import asyncio
+import click
+from remme import Remme
+
+from cli.blockchain_info.help import (
+    GET_BATCH_ID_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_START_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_LIMIT_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_HEAD_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_REVERSE_ARGUMENT_HELP_MESSAGE,
+)
+from cli.blockchain_info.service import BlockchainInfo
+from cli.constants import (
+    BATCH_ID_REGEXP,
+    FAILED_EXIT_FROM_COMMAND,
+    NODE_URL_ARGUMENT_HELP_MESSAGE,
+)
+
+loop = asyncio.get_event_loop()
+
+
+@click.group('batch', chain=True)
+def batch_commands():
+    """
+    Provide commands for working with batches.
+    """
+    pass
+
+
+@click.option('--id', type=str, required=True, help=GET_BATCH_ID_ARGUMENT_HELP_MESSAGE)
+@click.option('--node-url', type=str, help=NODE_URL_ARGUMENT_HELP_MESSAGE)
+@batch_commands.command('get-single')
+def get_single(id, node_url):
+    """
+    Get batch by id.
+    """
+    if re.match(pattern=BATCH_ID_REGEXP, string=id) is None:
+        click.echo('The following id `{batch_id}` is not valid.'.format(batch_id=id))
```

Comment:
`The following id` > `The following batch id`.

Reply:
Done.